src/components/CreateBook.js

```diff
@@ -1,14 +1,31 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
- [ ] Kindly remove this `Eslint` disabling comment. Removing `Eslint` helps us to write clean, scalable and industry-standard JavaScript codes and using an `Eslint` disabling comment defeats this purpose. 👍

Reply:
removed the linter disabler from all the files to follow best practice